torch_geometric/explain/algorithm/captum.py

```diff
@@ -44,7 +44,6 @@ def __init__(
         self.model_config = model_config
 
     def forward(self, mask, *args):
-        """"""
```

Comment:
Can we not drop these? They are important to hide the doc-string of `forward` from `torch.nn.Module` in our docs.